app/konnect/reference/search.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Konnect Search
```

Comment:
@lena-larionova @Guaris Hi! I moved this Search page from the API section to the reference section. Do you think it makes sense here, or would it be better in another section?